src/Core/Components/DataGrid/FluentDataGrid.razor

```diff
@@ -18,7 +18,7 @@
                           grid-template-columns=@_internalGridTemplateColumns
                           class="@GridClass()"
                           style="@Style"
-                          aria-rowcount="@(_ariaBodyRowCount + 1)"
+                          aria-rowcount="@(_internalGridContext.TotalItemCount + 1)"
```

Comment:
Why did you add one item: `+ 1` generates an invalid value.

Reply:
The +1 was already there. Did not change that with this PR